lib/bin_conf/Bin_conf.ml

```diff
@@ -293,19 +293,21 @@ let update_from_ocp_indent c (oic : IndentConfig.t) =
     | Never -> `Never
     | Auto -> `Auto
   in
+  let elt v = Conf.Elt.make v `Default in
```

Comment:
We should have a "from" for ocp-indent config.

Reply:
That sounds reasonable. I have no idea how this feature is supposed to work though, could you point me to an example ?

Reply:
I had a go at it in my last commit https://github.com/ocaml-ppx/ocamlformat/pull/2188/commits/49d99ac4ae325300866c88c57a8636911b2563f1 (there is no redundancy check as it would require more heavy lifting and it's not really a must-have for that), let me know what you think

Reply:
Nice :) Do you think we can get rid of `` `Default `` ?

Reply:
I don't think we can